vscode-dotnet-runtime-extension/src/extension.ts

```diff
@@ -36,6 +36,8 @@ import {
     registerEventStream,
     RuntimeInstallationDirectoryProvider,
     VersionResolver,
```

Comment:
per @/dsplaisted, we should also show the message about telemetry if it is enabled, because the first run message may get swallowed by CDK or C# extensions. We should add an env var for the future .NET SDKs and for CDK to pass to opt out of SDK Telemetry just for them, and also so we dont let the first run message get swallowed, so we can show SDK telemetry after that globally on the system outside of CDK or C# in vs code

Reply:
Done.